lib/standard/collection/union_find.nit

```diff
@@ -33,10 +33,46 @@ import hash_collection
 # The methods `in_same_subset`, `to_partitions`, and their variations are offered instead.
 class DisjointSet[E]
 	super SimpleCollection[E]
+	super Cloneable
 
 	# The node in the hiearchical structure for each element
 	private var nodes = new HashMap[E, DisjointSetNode]
 
+	# Copy constructor
+	init from(other: DisjointSet[E])
```

Comment:
Yes, but it is a feature. The from of collection is already a loosy copy
constructor. Cf PR comment

Le mar. 31 mars 2015 01:42, Alexandre Terrasa <notifications@github.com> a
écrit :

> In lib/standard/collection/union_find.nit
> <https://github.com/privat/nit/pull/1226#discussion_r27419390>:
>
> >
> >  	# The node in the hiearchical structure for each element
> >  	private var nodes = new HashMap[E, DisjointSetNode]
> >
> > +	# Copy constructor
> > +	init from(other: DisjointSet[E])
>
> from is already used by some classes to be constructed from other classes
> like Collection::from. I see a risk of confusion here.
>
> Maybe we should state once and for all the name of a copy constructor like
> copy?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/privat/nit/pull/1226/files#r27419390>.
>